.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,9 +6,10 @@
 
 **Review Checklist**
 I certify that I have:
-- [ ] tested my changes
-- [ ] added/updated automated tests
 - [ ] updated the changelog
+- [ ] manually tested my changes
+- [ ] added/updated automated unit/integration tests
+- [ ] created/ran system tests (provide build number)
```

Comment:
These changes look good, but should we remove the content in the parenthesis? The system tests can be run locally, so they might not have a build number.

Reply:
Alternatively add "if available"